code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -165,6 +165,10 @@ var/last_chew = 0
 /obj/item/weapon/handcuffs/cyborg
 	dispenser = 1
 
+/obj/item/weapon/handcuffs/cyborg/afterattack(atom/A, mob/user as mob, proximity)
```

Comment:
Shouldn't it be scooping up those handcuffs and refilling its own stock or something? 
just destroying clicked-on cuffs seems bizarre and mildly open to abuse

Reply:
it either can be slapped to synth system or stock system, but its complicated for this PR.
I can add this later, should i remove it now or leave how it is ? It's still better to give people free handcuffs